tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/TableHandle.java

```diff
@@ -99,6 +110,10 @@ public String getName()
         return schema;
     }
 
+    public boolean getShouldCreateSchema() {
```

Comment:
accessors for booleans should start from `is`. In this case I would get rid of `get` or renamed it to `isSchemaCreationNeeded()`